### PR TITLE
Make team TORCH the maintainers

### PR DESCRIPTION
**Pull request by prayerslayer**
This reflects reality better.

### CODE REVIEW

File: MAINTAINERS [hunk 1]
```diff
@@ -1,4 +1 @@
-Maximilian Fellner <maximilian.fellner@zalando.de>
-Nikolaus Piccolotto <nikolaus.piccolotto@zalando.de>
-Daniel Spaude <daniel.spaude@zalando.de>
-Oleksii Khomchenko <oleksii.khomchenko@zalando.de>
+Team TORCH <team-torch@zalando.de>
```
Comment (alexkops) [inline MAINTAINERS]: Can team mailinglists receive mails from external?

Comment (prayerslayer) [inline MAINTAINERS]: yes, i checked.

**Comment by LappleApple** [comment]
👍

**Comment by LappleApple** [comment]
@prayerslayer Want to approve your own change and then merge?

**Comment by prayerslayer** [comment]
👍

**Comment by prayerslayer** [comment]
Thanks

**Pull request merged**